src/script/conversation/linkPreviews/helpers.test.ts

```diff
@@ -73,10 +73,16 @@ describe('getFirstLinkWithOffset', () => {
     const singleTick = 'cool code: `wire.com`';
     const moreTicks = 'cool code: ```\nwire.com\n```';
     const manyTicks = 'cool code: ``````\nwire.com\n``````';
+    const multilineSingleTick = '\ncool code: `wire.com`';
+    const noClosingTicks = 'cool code: ```\nwire.com';
+    const ticksception = "cool code: ```\nwire.com `it's a trap!`\n```";
 
     expect(getFirstLinkWithOffset(singleTick)).toBeUndefined();
     expect(getFirstLinkWithOffset(moreTicks)).toBeUndefined();
     expect(getFirstLinkWithOffset(manyTicks)).toBeUndefined();
+    expect(getFirstLinkWithOffset(multilineSingleTick)).toBeUndefined();
+    expect(getFirstLinkWithOffset(noClosingTicks)).toBeUndefined();
+    expect(getFirstLinkWithOffset(ticksception)).toBeUndefined();
```

Comment:
This covers the 2 cases that were causing issues `noClosingTicks` and  `ticksception`, both cases fail with the previous implementation.
`multilineSingleTick` was added because I ran into that issue looking for a solution, this one does not fail with the previous implementation.